test/studies/champs/functions.bash

```diff
@@ -71,14 +94,21 @@ function test_compile() {
   fi
   test_end
 
-  $CHPL_HOME/util/test/computePerfStats comp-time-$kind $CHPL_TEST_PERF_DIR/$CHPL_TEST_PERF_DESCRIPTION $CHAMPS_GRAPH_PATH/comp-time.perfkeys $kind.comp.out.tmp
-  if [[ $? -ne 0 ]] ; then
-    log_fatal_error "computing compile time stats for ${kind}"
+  # don't leave this environment set
+  if [ ! -z "$CHAMPS_QUICKSTART" ]; then
+    unset CHPL_STOP_AFTER_PASS
   fi
 
-  $CHPL_HOME/util/test/computePerfStats emitted-code-size-$kind $CHPL_TEST_PERF_DIR/$CHPL_TEST_PERF_DESCRIPTION $CHAMPS_GRAPH_PATH/emitted-code-size.perfkeys $kind.comp.out.tmp
-  if [[ $? -ne 0 ]] ; then
-    log_fatal_error "computing emitted code size stats for ${kind}"
+  if [ -z "$CHAMPS_QUICKSTART" ]; then
+    $CHPL_HOME/util/test/computePerfStats comp-time-$kind $CHPL_TEST_PERF_DIR/$CHPL_TEST_PERF_DESCRIPTION $CHAMPS_GRAPH_PATH/comp-time.perfkeys $kind.comp.out.tmp
+    if [[ $? -ne 0 ]] ; then
+      log_fatal_error "computing compile time stats for ${kind}"
+    fi
+
+    $CHPL_HOME/util/test/computePerfStats emitted-code-size-$kind $CHPL_TEST_PERF_DIR/$CHPL_TEST_PERF_DESCRIPTION $CHAMPS_GRAPH_PATH/emitted-code-size.perfkeys $kind.comp.out.tmp
```

Comment:
This looks new. I'm assuming you added this on purpose (no problem if so, just want to make sure it's intentional since I didn't see measuring code size stats as something on the OP).

Reply:
Not new: https://github.com/chapel-lang/chapel/pull/23444/files/f63665ff36777ba4a4c590543c4a31df6c191423#diff-cf76251111f8b3e0db292e2815f620b5945b34a8b74ed91b3c75f6d87877a7a9L79. Just moved inside `if quickstart`.